dataprofiler/data_readers/data_utils.py

```diff
@@ -90,7 +105,7 @@ def unicode_to_str(data, ignore_dicts=False):
     # if data is a dictionary
     if isinstance(data, dict) and not ignore_dicts:
         return {
-            unicode_to_str(key, ignore_dicts=True): unicode_to_str(
+            cast(str, unicode_to_str(key, ignore_dicts=True)): unicode_to_str(
```

Comment:
there's no guarantee this is a string, this could be an int, right?

Reply:
Removed the cast.

Reply:
awesome -- thx, @Sanketh7

Reply:
is it removed @Sanketh7 ?  looks like it is still there ...

Reply:
Should be removed by https://github.com/capitalone/DataProfiler/pull/662/commits/8de0ceff3ff0503ae4422de6c2b804700e34f484